tests/utilities/nidmm_measurement/__init__.py

```diff
@@ -0,0 +1,109 @@
+"""NI-DMM MeasurementLink test service."""
+import math
+import pathlib
+from enum import Enum
+from typing import Iterable, Sequence, Tuple
+
+import nidmm
+
+import ni_measurementlink_service as nims
+
+
+service_directory = pathlib.Path(__file__).resolve().parent
+measurement_service = nims.MeasurementService(
+    service_config_path=service_directory / "NIDMMMeasurement.serviceconfig",
+    version="0.1.0.0",
+    ui_file_paths=[
+        service_directory,
+    ],
+)
+
+
+class Function(Enum):
```

Comment:
Do we really need to create this class? I see this is only used in https://github.com/ni/measurementlink-python/pull/504/files#diff-a97403d3637222c25a83c1fb125bed1727aeb8be60e2b9c8e99846c331d748a0R95